config/core/test-configs.yaml

```diff
@@ -2335,14 +2356,17 @@ test_configs:
   - device_type: r8a774a1-hihope-rzg2m-ex
     test_plans:
       - baseline
+      - baseline-cip-nfs
```

Comment:
There's a problem with this one: https://lava.ciplatform.org/scheduler/job/512994

Reply:
Tested OK with stable 5.4: https://staging.kernelci.org/test/plan/id/61890f6e98d5806260a6c438/

So there might be an issue in linux-next with this platform.  It's not a problem for merging this PR though.